bytes/bytes/database/sql_meta_repository.py

```diff
@@ -252,6 +252,8 @@ def to_boefje_meta_in_db(boefje_meta: BoefjeMeta) -> BoefjeMetaInDB:
         organization=boefje_meta.organization,
         started_at=boefje_meta.started_at,
         ended_at=boefje_meta.ended_at,
+        runnable_hash=boefje_meta.runnable_hash,
+        environment=boefje_meta.environment,
```

Comment:
These should be added in `to_boefje_meta` as well.

Reply:
Done